modules/images/webp-uploads/load.php

```diff
@@ -31,6 +31,11 @@ function webp_uploads_filter_image_editor_output_format( $output_format, $filena
 		return $output_format;
 	}
 
+	// Skip conversion when creating the `-scaled` image (for large image uploads).
+	if ( 1 === preg_match( '/.*-scaled\..{3}.?$/', $filename ) ) {
+		return $output_format;
+	}
```

Comment:
Can we add a test for this, with a couple different file names?

Reply:
Added in [`98e1003` (#143)](https://github.com/WordPress/performance/pull/143/commits/98e100336738c0fd4b1ad8c56536b10d1a3825fc)